README.md

```diff
@@ -108,16 +108,37 @@ skew may result in `system time has changed` messages in VM logs.
 #### Network
 
 The guest agent uses network interface metadata to manage the network
-interfaces in the guest by performing the following tasks:
-
-*   Enabled all associated network interfaces on boot.
+interfaces by performing the following tasks:
+
+*   Enable all associated network interfaces on boot.
+    *   Detect the current active network manager service that is managing the
+        primary NIC.
+    *   Rollback and delete any files/configurations left behind by all other
+        supported network manager services.
+    *   Write and apply new configurations for the secondary NICs using the
+        network manager service detected in the first step.
 *   Setup or remove IP routes in the guest for IP forwarding and IP aliases
     *   Only IPv4 IP addresses are currently supported.
     *   Routes are set on the primary ethernet interface.
     *   Google routes are configured, by default, with the routing protocol ID
         `66`. This ID is a namespace for daemon configured IP addresses. It can
         be changed with the config file, see below.
 
+On Linux, supported network managers are:
+
+*   `systemd-networkd`
+*   `wicked`
+*   `NetworkManager`
+*   `netplan`
+
+If none of these network interfaces are detected on the system, then the agent
+will default to using `dhclient` for managing network interfaces. The agent will
+not manage the primary NIC unless `manage_primary_nic` is set to `true` in the
```

Comment:
```
The agent will
not manage the primary NIC unless `manage_primary_nic` is set to `true` in the
instance configuration.
```

Is this statement in the context of dhclient or is it a univeral statement?

If it's a universal statement this I'd suggest using a non-paragraph approach to each that each statement can be read as standing alone and of equal merit.

I'd also suggest that "the following features depend on `manage_primary_nic` being enabled. 
* feature abc: This enables foo, etc.

Reply:
Non-paragraph-ized this section and made the blurb about the configuration flag more clear.